tests/ui/coherence/orphan-check-projections-dont-cover.rs

```diff
@@ -0,0 +1,26 @@
+// Regression test for issue #99554.
+// Don't consider projections to cover type parameters.
+
+// revisions: classic next
+//[next] -Ztrait-solver=next
```

Comment:
```suggestion
//[next] compile-flags: -Ztrait-solver=next
```

Oopsie :sweat_smile: